SwanContents/swancontents/filemanager/swan_cs3_filemanager.py

```diff
@@ -0,0 +1,41 @@
+from .projects_mixin import ProjectsMixin
+from cs3api4lab import CS3APIsManager
+
+class SwanCs3FileManager(ProjectsMixin, CS3APIsManager):
```

Comment:
I am hesitating whether we should include this now, if it's just an empty shell. We could just do the refactoring to make the current filemanager the eos file manager, that is good, but the Cs3FileManager inclusion is just adding a skeleton without real functionality. This should be delayed until there is real functionality.